os/android/platformdefines.h

```diff
@@ -32,6 +32,10 @@
 
 #include <ui/GraphicBuffer.h>
 
+#ifdef USE_VK
+#include <gbm.h>
```

Comment:
@kalyankondapally It seems you comment at an obsolete patch, this code is not exit in the latest patch.